packages/datadog-plugin-mocha/test/mocha-test-integration-http.js

```diff
@@ -2,10 +2,12 @@ const http = require('http')
 
 describe('mocha-test-integration-http', () => {
   it('can do integration http', (done) => {
-    const req = http.request('http://test:123', (res) => {
-      expect(res.statusCode).to.equal(200)
-      done()
-    })
-    req.end()
+    setTimeout(() => {
```

Comment:
this is just to test that everything works even if the request is not fired synchronously with the start of the test